Onboarding/TechicalOverview.md

```diff
@@ -0,0 +1,30 @@
+# CanDIG Technical Onboarding
+## CanDIG Project Overview
+The CanDIG project has a distributed architecture, meaning each site is meant to run their own _____(What do you call everything represented in the diagram together) and there is no centralized CanDIG server. Users can use their local CanDIG _____(^) to communicate with other sites. Every site is responsible for handling the authentication of it's users and the authorization of incoming requests from other sites. CanDIG uses an API driven approach, where users send requests to their CanDIG which communicate with other sites through ...?
+## Component Overview:
+### Tyk [Gateway, Pump]
+Tyk Gateway is an API managment platform that CanDIG uses to map API endpoints to endpoints in CanDIG server. When the user makes a request, it is made to Tyk, which forwards the request to the mapped endpoint in CanDIG server.  
+Tyk Pump collects metrics on the use of Gateway and has many methods for transmiting these metrics to various databases.
+### Keycloak
+Authentication. That's it.?
+### CanDIG server
+Fasciliates the communication between different sites running CanDIG. What else?
+### mongodb, redis
+Databases. What do they store? Absolutely no idea. Maybe this is what pump goes to?
+
+## How they interact:
+![Architecture Diagram](Figures/ArchitectureDiagram.png)
```

Comment:
Great job at diagramming! I really liked the solid line between the CanDIG server and other CanDIG sites and dashed otherwise. Looking at it, I think we need to use a different icon for other CanDIG sites. It is confusing as a reader to think that other sites do not have the remaining tools like Tyk etc. What do you think?

Reply:
That's a very good point, I think putting a box around the site and then representing the other sites as a similarly labeled box would be better

Reply:
Love the idea! Go for it!